backend/src/main/resources/schema.graphqls

```diff
@@ -46,7 +46,7 @@ type Patient {
   lastName: String
   suffix: String
   race: String
-  birthDate: String
+  birthDate: LocalDate
```

Comment:
Where does `LocalDate` get defined for the purposes of graphql? From reading [this article](https://medium.com/supercharges-mobile-product-guide/graphql-server-using-spring-boot-part-ii-scalars-31505fe90c4c) it seems like we need additional `GraphQLScalarType` plumbing to define it.

Reply:
It's declared up top in this file (just the statement `scalar LocalDate`; it's defined by the graphql-datetime-autoconfigure plugin, as configured in [application.yaml](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/resources/application.yaml#L27).

Reply:
I just had to restart the backend, I was getting an error about LocalDate otherwise. I'm assuming there's no "watch" task that will ensure I do that?

Reply:
It's possible I think, but I'm not sure how to combine it with your docker build.